api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.nicknathanjustin.streamercontracts.contracts;
+
+import lombok.RequiredArgsConstructor;
+import lombok.NonNull;
+import org.springframework.lang.Nullable;
+
+import java.util.Calendar;
+import java.util.UUID;
+import java.sql.Timestamp;
+
+@RequiredArgsConstructor
+public class ContractServiceImpl implements ContractService {
+
+    @NonNull final ContractModelRepository contractModelRepository;
+
+    @Override
+    public ContractModel createContract(@NonNull UUID proposerId, @NonNull UUID streamerId, @Nullable String game, @NonNull String description) {
+        final Timestamp creationTimestamp = new Timestamp(System.currentTimeMillis());
+        // Add 10 days to the current timestamp to create the expires timestamp
+        final Calendar cal = Calendar.getInstance();
```

Comment:
super-super-optional: Im a fan of spelling out the word. I personally would rather read "calendar" than a shortened "cal"... I realize this is my personal preference and not something ill force

Reply:
Think I was copying this from stack overflow or something, I'm a fan of writing it out as well.